MachineLearning_Engine/Compute/LinearRegression.py

```diff
@@ -41,6 +41,8 @@ def fit(x: np.ndarray, y: np.ndarray):
 	model.fit(x, y)
 	return model
 
+def coefs(model: LinearRegression):
```

Comment:
If you get the attributes directly as in the comment above, we can also simplify by removing this, as it becomes redundant!